its/src/test/java/com/sonar/it/scanner/msbuild/sonarqube/ScannerMSBuildTest.java

```diff
@@ -1415,29 +1418,30 @@ private void runCSharpSharedFileWithOneProjectUsingProjectBaseDir(Function<Path,
   private BuildResult runAnalysisWithoutProjectBasedDir(Path projectDir) {
     String token = TestUtils.getNewToken(ORCHESTRATOR);
     String folderName = projectDir.getFileName().toString();
-    ScannerForMSBuild scanner = TestUtils.newScanner(ORCHESTRATOR, projectDir, ScannerClassifier.NET, token)
+    ScannerCommand scanner = TestUtils.newScannerBegin(ORCHESTRATOR, folderName, projectDir, token, ScannerClassifier.NET)
       .addArgument("begin")
       .setProjectKey(folderName)
       .setProjectName(folderName)
       .setProjectVersion("1.0")
-      // do NOT set "sonar.projectBaseDir" for this test
+      .setProperty("sonar.projectBaseDir", null)  // Do NOT set "sonar.projectBaseDir" for this test. We need to remove the default value
       .setScannerVersion(TestUtils.developmentScannerVersion())
       .setEnvironmentVariable(AzureDevOpsUtils.ENV_SOURCES_DIRECTORY, "")
       .setProperty("sonar.verbose", "true")
       .setProperty("sonar.sourceEncoding", "UTF-8");
 
-    ORCHESTRATOR.executeBuild(scanner);
+    scanner.execute(ORCHESTRATOR);
 
     BuildResult buildResult = TestUtils.runDotnetCommand(projectDir, "build", folderName + ".sln", "--no-incremental");
 
     assertThat(buildResult.getLastStatus()).isZero();
 
     // use executeBuildQuietly to allow for failure
-    return ORCHESTRATOR.executeBuildQuietly(TestUtils.newScanner(ORCHESTRATOR, projectDir, ScannerClassifier.NET, token)
+    return TestUtils.newScannerEnd(ORCHESTRATOR, projectDir, ScannerClassifier.NET, token)
```

Comment:
There are a few instanced where `ORCHESTRATOR.executeBuildQuietly` was used instead of the more common  `executeBuild`. I don't see how this difference was preserved. Is this a safe change?

Reply:
Yes, the only different is amount of logs produced.
I don't know why the quiet build was used. If anything fails, I would like to see the logs. So I changed it to non-quiet everywhere.